app.bbsync.BlackboardSync.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Jacob Sanchez -->
+<component type="desktop-application">
+  <id>app.bbsync.BlackboardSync</id>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <content_rating type="oars-1.1" />
+  <launchable type="desktop-id">app.bbsync.BlackboardSync.desktop</launchable>
+  <name>BlackboardSync</name>
+  <summary>Download your Blackboard Learn content to your device.</summary>
+  <summary xml:lang="es">Descarga contenido de Blackboard Learn a tu dispositivo.</summary>
+
+  <description>
+    <p>A student favourite, now available on FlatHub!</p>
+    <p>Download all your content from Blackboard Learn to your device, without any intervention.</p>
+    <p>Sync content like text documents, spreadsheets and lecture slides, which you can later access offline.</p>
+    <p>More than 30 universities supported around the world.</p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://bbsync.app/assets/screenshots/BBS_Tray.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://bbsync.app/assets/screenshots/BBS_Login.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://bbsync.app/assets/screenshots/BBS_Notification.png</image>
+    </screenshot>
+  <screenshots/>
+
+  <url type="homepage">https://github.com/jacobszpz/BlackboardSync</url>
+</component>
```

Comment:
missing required `releases`, `content_rating`

Reply:
content rating is there, at the top. I will add releases